rktl_sim/src/simulator/car.py

```diff
@@ -104,23 +122,53 @@ def get_pose(self, noise=None):
                 pos = np.random.normal(pos, noise['pos'])
                 orient = np.random.normal(orient, noise['orient'])
 
-        return (pos, p.getQuaternionFromEuler(orient))
+        return pos, p.getQuaternionFromEuler(orient)
 
     def get_velocity(self):
+        """
+        @return: the velocity of the car
+        """
         link_state = p.getLinkState(self.id, self.body_link_id, computeLinkVelocity=1)
         orient = link_state[1]
         linear, angular = link_state[6:8]
         heading = p.getEulerFromQuaternion(orient)[2]
         r_inv = np.array([[math.cos(heading), -math.sin(heading), 0.],
                           [math.sin(heading), math.cos(heading), 0.],
-                         [0., 0., 1.]], dtype=np.float)
+                          [0., 0., 1.]], dtype=np.float)
         linear = r_inv @ linear
-        return (linear, angular)
+        return linear, angular
 
     def reset(self, pos, orient):
+        """
+        uses the position and the orientation to reset system state and car configuration
```

Comment:
In other comments, we use capitalization and punctuation, we should strive for consistency

Reply:
I worked through all of the files and added capitalization, punctuation, and fixed grammar along the way. I think that I covered all of the files, but I'm not sure. Lmk if there are some that I missed?